release_notes.md

```diff
@@ -0,0 +1,3 @@
+ # Release notes
```

Comment:
Should we have a description of format here? What and how should I put into this file during the release process?

We had a description with the purpose of the file for the CHANGELOG, but it seems missing here?

Reply:
Thanks, fixed it.